### PR TITLE
Assume tests container is started immediately

### PR DESCRIPTION
**Pull request by kokosing**
Assume tests container is started immediately

### CODE REVIEW

File: presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java [hunk 1]
```diff
@@ -72,6 +72,7 @@
     private static final Logger log = Logger.get(TestRun.class);
 
     @Option(names = {"-h", "--help"}, usageHelp = true, description = "Show this help message and exit")
+    @SuppressWarnings("unused")
```
Comment (findepi) [inline presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java:73]: @wendigo can we remove the field?

File: presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java [hunk 2]
```diff
@@ -276,10 +274,7 @@ private Environment getEnvironment()
                                                 .build()))
                                 .addAll(testArguments)
                                 .addAll(reportsDirOptions(reportsDirBase))
-                                .build().toArray(new String[0]))
-                        // this message marks that environment has started and tests are running
-                        .waitingFor(forLogMessage(".*\\[TestNG] Running.*", 1)
-                                .withStartupTimeout(ofMinutes(15)));
+                                .build().toArray(new String[0]));
```
Comment (findepi) [inline presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java:277]: Add rationale to commit message and/or the code.

**Comment by kokosing** [comment]
@findepi AC, PTAL

**Review by findepi — APPROVED** [review]

**Comment by findepi** [comment]
Thanks for taking care of this

**Review by Praveen2112 — APPROVED** [review]

**Review by zaz968m — APPROVED** [review]

**Pull request merged**